pages/docs/security.mdx

```diff
@@ -0,0 +1,114 @@
+---
+description: Survey of common security problems facing LLM-based applications and how to use Langfuse to trace, prevent, and evaluate security risks.
+---
+
+# Security with Langfuse
+
+There are a host of potential security risks involved with LLM-based applications, such as prompt injection, personally identifiable information (PII), or harmful prompts. Langfuse provides a comprehensive and easy way to monitor and protect against these security risks.
+
+This document will provide an overview of how Langfuse can be used with security tools. In the example below we use a [LLM Guard](https://llm-guard.com/), but there are many open-source and/or paid security tools available, such as Prompt Armor, Microsoft Azure Responsible AI, and Lakera Guard.
```

Comment:
In the example below we use the open source library [LLM Guard](https://llm-guard.com/), but there are other open-source and/or paid security tools available, such as Prompt Armor, Nemo Guardrails, Microsoft Azure Responsible AI, and Lakera Guard